<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>171</height>
       </size>
      </property>
      <widget class="QLabel" name="scorePlayer2Label">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>0</y>
         <width>81</width>
         <height>121</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/P2 - 0pt.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="scorePlayer1Label">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>0</y>
         <width>81</width>
         <height>121</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/P1 - 0pt.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="updatePlayer2PicButton">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>10</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border-image: url(:img/GUI/updateprofile1.png);
background: none;
border: none;
background-repeat: none;
color : white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="player2Label">
       <property name="geometry">
        <rect>
         <x>1180</x>
         <y>0</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : black;</string>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/PP.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>1050</x>
         <y>10</y>
         <width>108</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Player 2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>50</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>331</width>
         <height>71</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:img/GUI/rightLifeBar.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="player2Lifebar">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>70</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
     background-color: #FF0000;
 }</string>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>331</width>
         <height>71</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:img/GUI/leftLifeBar.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QProgressBar" name="player1Lifebar">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
     background-color: #FF0000;
 }</string>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="player1Label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/PP.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </widget>
      <widget class="QPushButton" name="updatePlayer1PicButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border-image: url(:img/GUI/updateprofile2.png);
background: none;
border: none;
background-repeat: none;
color : white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>-30</y>
         <width>108</width>
         <height>130</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Player 1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="playPauseButton">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>30</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border-image: url(:img/GUI/pause.png);
background: none;
border: none;
background-repeat: none;
</string>
       </property>
       <property name="text">
        <string>Press to play !</string>
       </property>
      </widget>
      <widget class="QPushButton" name="updateBGButton">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>570</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border-image: url(:img/GUI/updateBG.png);
background: none;
border: none;
background-repeat: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ultimateBarLabel">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>550</y>
         <width>491</width>
         <height>111</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/Ultbar.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="quitButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>560</y>
         <width>90</width>
         <height>90</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border-image: url(:img/GUI/quitter.png);
background: none;
border: none;
background-repeat: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="outlineLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>140</y>
         <width>761</width>
         <height>411</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/frame3.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="cameraLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>721</width>
         <height>391</height>
        </rect>
       </property>
       <property name="text">
        <string>Camera Placeholder</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="Line" name="lineP1">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>361</width>
         <height>391</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="activePlayer1">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>351</width>
         <height>391</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="activePlayer1_timer">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>361</width>
         <height>391</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="activePlayer2">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>151</y>
         <width>351</width>
         <height>391</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="activePlayer2_timer">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>150</y>
         <width>351</width>
         <height>391</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="Line" name="lineP2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>140</y>
         <width>371</width>
         <height>401</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="seperatingLine">
       <property name="geometry">
        <rect>
         <x>599</x>
         <y>140</y>
         <width>71</width>
         <height>401</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(185, 187, 173)</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QProgressBar" name="ultProgressBar">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>580</y>
         <width>431</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> QProgressBar::chunk {
     background-color: #3DAAFF;
 }</string>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="backgroundLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>-5000</x>
         <y>-10</y>
         <width>1271</width>
         <height>701</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/BG1.PNG</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="titleLabel">
       <property name="geometry">
        <rect>
         <x>264</x>
         <y>145</y>
         <width>751</width>
         <height>401</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="startingScreenBG">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>-16</x>
         <y>-25</y>
         <width>1291</width>
         <height>721</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/img/GUI/MarvelStyle.png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="startingButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>540</y>
         <width>311</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border-image: url(:img/GUI/pause.png);
background: none;
border: none;
background-repeat: none;
color : black;</string>
       </property>
       <property name="text">
        <string>Start to Play !</string>
       </property>
      </widget>
      <zorder>backgroundLabel</zorder>
      <zorder>ultProgressBar</zorder>
      <zorder>scorePlayer2Label</zorder>
      <zorder>scorePlayer1Label</zorder>
      <zorder>updatePlayer2PicButton</zorder>
      <zorder>player2Label</zorder>
      <zorder>label_4</zorder>
      <zorder>player2Lifebar</zorder>
      <zorder>player1Lifebar</zorder>
      <zorder>player1Label</zorder>
      <zorder>updatePlayer1PicButton</zorder>
      <zorder>label_3</zorder>
      <zorder>playPauseButton</zorder>
      <zorder>updateBGButton</zorder>
      <zorder>ultimateBarLabel</zorder>
      <zorder>quitButton</zorder>
      <zorder>cameraLabel</zorder>
      <zorder>lineP1</zorder>
      <zorder>activePlayer1</zorder>
      <zorder>activePlayer1_timer</zorder>
      <zorder>activePlayer2</zorder>
      <zorder>activePlayer2_timer</zorder>
      <zorder>lineP2</zorder>
      <zorder>seperatingLine</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>outlineLabel</zorder>
      <zorder>startingScreenBG</zorder>
      <zorder>titleLabel</zorder>
      <zorder>startingButton</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>754</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>738</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
